seeds/connections-seed.js

```diff
@@ -1,17 +1,78 @@
 // Currently based of statuses, variants, should cover all possible scenarios within table boundries
 
-const company_name = 'Seeded company_name Name'
-const company_number = '0000000'
-const variants = [
-  { status: 'pending', company_number: company_number + '0', company_name },
-  { status: 'pending', company_number: company_number + '1', company_name: '_@@$@*@&%£*&@£*$*(@£*$)@£*&' },
-  { status: 'unverified', company_number: company_number + '2', company_name: 0 },
-  { status: 'unverified', company_number: company_number + '3', company_name },
-  { status: 'verified_them', company_number: company_number + '4', company_name },
-  { status: 'verified_us', company_number: company_number + '5', company_name },
-  { status: 'verified_both', company_number: company_number + '6', company_name },
-  { status: 'disconnected', company_number: company_number + '7', company_name },
-]
+// const company_name = 'Seeded company_name Name'
+// const company_number = '0000000'
+// 10 000 rows, unique comp. nos - change them to start with a 1, interpret as a number, increment and change to stringon insert, don't worry about changing the company name for now...maybe set one to a unique name
+// const variants = [
+//   { status: 'pending', company_number: company_number + '0', company_name },
+//   { status: 'pending', company_number: company_number + '1', company_name: '_@@$@*@&%£*&@£*$*(@£*$)@£*&' },
+//   { status: 'unverified', company_number: company_number + '2', company_name: 0 },
+//   { status: 'unverified', company_number: company_number + '3', company_name },
+//   { status: 'verified_them', company_number: company_number + '4', company_name },
+//   { status: 'verified_us', company_number: company_number + '5', company_name },
+//   { status: 'verified_both', company_number: company_number + '6', company_name },
+//   { status: 'disconnected', company_number: company_number + '7', company_name },
+// ]
+
+export async function prepareVariants() {
+  const variants10000 = []
+  for (let i = 0; i < 10000; i++) {
```

Comment:
can we make the `10000` a const at the top of the file and then make you branches conditional on `i % 7`. That way we can adjust the import size more easily